### PR TITLE
New dashboard

### PR DESCRIPTION
**Pull request by soupette**

### CODE REVIEW

**Comment by mnlbox** [comment]
@soupette can you add some screen shot for this PR please?
I'm so exited about this :wink: :heart:

**Comment by soupette** [comment]
Hi @mnlbox !
Here's a preview
screen shot 2018-04-05 at 10 09 43 am

**Comment by mnlbox** [comment]
@soupette Thanks. Server information can be very useful :pray:

**Comment by soupette** [comment]
@mnlbox server's info won't be implemented yet

**Comment by dsibaja-dnamyc** [comment]
Wow when is this coming out?

File: packages/strapi-admin/admin/src/translations/fr.json [hunk 1]
```diff
@@ -9,10 +9,25 @@
...
   "app.components.HomePage.welcome": "Bienvenue à bord!",
-  "app.components.HomePage.description.part1": "Nous somme heureux de vous compter parmi nos utilisateurs",
-  "app.components.HomePage.description.part2": "Vous faites désormais parti de notre communauté qui peut vous aider à développer votre application.",
-  "app.components.HomePage.button": "Créez votre premier type de contenu",
-  "app.components.HomePage.feedback": "N'hésitez pas à utiliser un des channels ci-dessous pour poser vos questions ou nous donner vos retours.",
+  "app.components.HomePage.cta": "CONFIRMEZ",
+  "app.components.HomePage.community": "Rejoignez la communauté",
+  "app.components.HomePage.community.content": "Discutez avec les membres de l'équipe, contributeurs et développeurs sur différent supports.",
+  "app.components.HomePage.create": "Créez votre premier Content Type",
+  "app.components.HomePage.welcomeBlock.content": "Nous sommes heureux de vous compter parmi nos membres. Nous sommes à l'écoute de vos retours alor, n'hésitez pas à nous envoyer des DM sur\u0020",
```
Comment (Aurelsicoko) [inline packages/strapi-admin/admin/src/translations/fr.json]: `s` missing at `alor`

File: packages/strapi-admin/admin/src/translations/fr.json [hunk 2]
```diff
@@ -9,10 +9,25 @@
...
+  "app.components.HomePage.welcomeBlock.content.issues": "issues",
+  "app.components.HomePage.welcomeBlock.content.raise": "\u0020ou soummetez des\u0020",
```
Comment (Aurelsicoko) [inline packages/strapi-admin/admin/src/translations/fr.json]: `soumettez`

File: packages/strapi-admin/admin/src/translations/fr.json [hunk 3]
```diff
@@ -9,10 +9,25 @@
...
+  "app.components.HomePage.createBlock.content.first": "Le\u0020",
+  "app.components.HomePage.createBlock.content.second": "\u0020plugin vous permet de définir la structure de vos modèles. Nous vous conseillons fortement de suivre notre\u0020",
+  "app.components.HomePage.createBlock.content.tutorial": "\u0020tutoriel.",
+  "app.components.HomePage.button.quickStart": "VOIR LE QUICK START TUTORIEL",
+  "app.components.HomePage.support": "SUPPORT US",
+  "app.components.HomePage.support.content": "En achetant notre T-shirt (25€), vous nous aidez a poursuivre à maintenir le projet pour que nouis puissons vous donner la meilleure expérience possible!",
```
Comment (Aurelsicoko) [inline packages/strapi-admin/admin/src/translations/fr.json]: `nous aidez à`

**Comment by nurikabe** [comment]
@soupette Please sell hats as well!

**Comment by ChristopherJohnson25** [comment]
What's the timeline on this update? Looks great!

**Pull request merged**